src/lib.rs

```diff
@@ -92,6 +92,27 @@
 //! }
 //! ```
 //!
+//! # Available logging frameworks
+//!
+//! In order to produce log output executables have to utilize
+//! a framework compatible with the facade.
```

Comment:
Almost none of these are a "framework." Maybe only `log4rs`. Something like `env_logger` is definitely not a "framework." I think above it refers to these using the word "implementation."

> The `log` crate provides a single logging API that abstracts over the actual logging implementation.